doc/example-spouse.md

```diff
@@ -165,7 +170,7 @@ Note that we declare a compound key of `(doc_id, sentence_index)` for each sente
 -->
 
 Next we declare a DDlog function which takes in the `doc_id` and `content` for an article and returns rows conforming to the sentences schema we just declared, using the **user-defined function (UDF)** in `udf/nlp_markup.sh`.
-This UDF is a Bash script which calls [our own wrapper around CoreNLP](https://github.com/HazyResearch/bazaar/tree/master/parser).
+This UDF is a Bash script which calls [our own wrapper around CoreNLP](https://github.com/HazyResearch/bazaar/tree/master/parser). The CoreNLP library requires Java 8 to run.
```

Comment:
It'd be nice to mention this Java 8 requirement in the first section for preparation/installation as well as in the quickstart, perhaps pointing to this page? http://www.oracle.com/technetwork/java/javase/downloads/jre8-downloads-2133155.html